lib/modules/versioning/hashicorp/index.ts

```diff
@@ -25,14 +26,15 @@ export function isValid(input: string): boolean {
     try {
       return npm.isValid(hashicorp2npm(input));
     } catch (err) {
+      logger.debug({ value: input }, 'Unsupported hashicorp versioning value');
       return false;
     }
   }
   return false;
 }
 
 function matches(version: string, range: string): boolean {
-  return npm.matches(version, hashicorp2npm(range));
+  return isValid(range) && npm.matches(version, hashicorp2npm(range));
```

Comment:
are we doing this at other versionings too? 🤔

Reply:
we don't seem to have the same problem with others, maybe because they don't throw

Reply:
I nearly added this here, but thought, its probably called before `matches` is called, as none of the other versioning do this